_inc/client/at-a-glance/activity.jsx

```diff
@@ -6,8 +6,7 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import DashItem from 'components/dash-item';
 import { translate as __ } from 'i18n-calypso';
-// import get from 'lodash/get';
-// import includes from 'lodash/includes';
+// import { get, includes } from 'lodash';
```

Comment:
why keep this?

Reply:
Seems out of scope, presumably was kept for a reason. I'm fine either way.

Reply:
From #8311 - seems like a pending todo that's waiting on rewind.

Reply:
Oh, be sure to look at the rest of the file. This is part of a bunch of code that was commented out 🙂